server/index.js

```diff
@@ -24,6 +24,31 @@ app.get('/api/images/:restaurantId', (req, res) => {
     });
 });
 
+app.post('/api/images', (req, res) => {
+  const data = req.body;
+  db.insert(data)
+    .then(res.status(201).end())
+    .catch(err => console.log('Error occurs in Post'))
+});
+
+app.put('/api/images/:restaurantId', (req, res) => {
+  const data = req.body;
+  const id = req.params.restaurantId;
+  console.log("=============");
+  console.log(data);
+  console.log(id);
+  db.updateData(id, data)
+    .then(res.status(201).end())
+    .catch(error => console.log('Error occurs in Put'))
```

Comment:
If an error occurs, the client still needs a res sent. Sometimes the error occurs just due to a timeout and cannot be detected during development. Your client will break if you do not return a response in a set time.

Reply:
You are right. It makes sense. Just modified it. Thanks!